src/listview.cpp

```diff
@@ -874,36 +874,52 @@ QString ListView::refNameAt(const QPoint &pos)
 	return QString();
 }
 
+/*
+ * Return the device pixel ratio
+ */
+const qreal ListViewDelegate::dpr(void) const {
+#if QT_VERSION >= QT_VERSION_CHECK(5,6,0)
+    return qApp->devicePixelRatio();
+#else
+    return 1.0;
+#endif
+}
+
 void ListViewDelegate::addTextPixmap(QPixmap** pp, SCRef txt, const QStyleOptionViewItem& opt) const {
 
 	QPixmap* pm = *pp;
-	int ofs = pm->isNull() ? 0 : pm->width() + 2;
-        int spacing = 4;
-        QFontMetrics fm(opt.font);
-        int pw = fm.boundingRect(txt).width() + 2 * spacing;
-        int ph = fm.height();
 
-    QSize pixmapSize(ofs + pw, ph);
+	const unsigned int mark_spacing = 2; // Space between markers in pixels
```

Comment:
It is highly preferable to avoid mixing functional changes and whitespace or code cleanup (like changing variable names without changing their use).